### PR TITLE
fix: update welcome screen to reflect new designs

### PR DESCRIPTION
**Pull request by sracca**
This PR adds missing icons to the welcome screen, as well as adds the fonts necessary to the welcome screen, and updates it to reflect the changes in design 
screen shot 2018-08-15 at 10 22 50 am

### CODE REVIEW

File: src/lib/theme.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const spacing = {
   xl: '40px',
   xxl: '52px',
   xxxl: '96px',
-  xxxxl: '240px',
+  xxxxl: '104px',
```
Comment (mulias) [inline src/lib/theme.js]: Not sure about this change -- if the `240px` value is already being used somewhere else then we should keep it. Also, having the difference between `xxxl` and `xxxxl` be 8 pixels seems a bit silly. What if we either hard coded `104px` for the welcome screen margins, or just used `xxxl` and not tell anyone that we skimped on the last 8 pixels?

Comment (geraldalewis) [reply src/lib/theme.js]: Agree with @mulias -- also, as the number of `x`s increases, it gets harder to tell one value from another (e.g., `xxxxxxxxxxxxxl` vs `xxxxxxxxxxxxxxl`). I advocate for removing the `xxxxl` value.

File: src/components/name-input.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  WhiteArrowIcon,
+} from '../styles/pages/welcome';
+import ArrowIcon from '../icons/white-arrow.png';
+
+export const NameInput = ({ nextPage }) => (
+  <NameInputView style={{ flex: 1, flexDirection: 'row' }}>
+    <TextInputWrapper>
+      <NameInputText placeholder="What&apos;s your name?" />
+    </TextInputWrapper>
+    <NextButton onPress={() => nextPage()}>
```
Comment (geraldalewis) [inline src/components/name-input.js]: What happens if we change this to `onPress={nextPage}`? Unless the handler cares about the arity/args, we should be able to get away with not creating an inline callback. Not necessary if it's not a quick change.

File: src/containers/Welcome/index.js [hunk 3]
```diff
@@ -34,13 +30,20 @@ export default class Welcome extends React.Component {
...
         <BottomAnchoredBackgroundImage />
         <View style={{ flex: 1 }}>
-          <Heading>Sign in here.</Heading>
+          <Image
+            style={{
+              alignSelf: 'flex-end',
+              marginRight: 44,
+              marginTop: 42,
+            }}
+            /* eslint-disable-next-line */
```
Comment (geraldalewis) [inline src/containers/Welcome/index.js]: Is this `eslint-disable` still necessary?

File: src/lib/theme.js (hunk 1 above)
Comment (jennyckaplan) [reply src/lib/theme.js]: ok sounds good, the 240px was used in the last Welcome screen design so it is not used anywhere else, but I agree the `xxxxx`'s become messy fast

**Comment by jennyckaplan** [comment]
@cbarber @mulias @geraldalewis new changes

**Comment by cbarber** [comment]
:shipit:

**Pull request merged**